### DISCUSSION TITLE
Bring back instant filtering option

### DISCUSSION THREAD

**Discussion by raags** [apache/superset#19092] [General]
It was removed due to performance reasons https://github.com/apache/superset/pull/15365 -- any plans to bring it back?

**Comment by villebro** [comment]
@raags the long term plan is to bring it back as a global option per dashboard - either all filters are instant, or none are. Unfortunately I don't think any committers are working on this right now so not sure when this will be implemented, but I'm happy to review a PR that reintroduces this as a dashboard property.

**Reply by raags** [reply]
A global option would be fine for my use case.

**Comment by xavier-GitHub76** [comment]
This disccusion could be closed. Subject already existing (#20663)
